ruby/alias_manager.rb

```diff
@@ -0,0 +1,224 @@
+=begin PSEUDOCODE
+  - Get name
+  - Convert name to array
+  - Swap first and last name
+    - Split by ' '
+    - Swap using helper variable (Is there a cleaner way?) (Used reverse)
+      - helper = [0]
+      - [0] = [1]
+      - [1] = helper
+  - Split names into individual letters
+  - Change all vowels (a, e, i, o, or u) to next vowel in sequence (loop)
+    - Detect if a letter is a vowel 
+      - If vowel, map next vowel in sequence
+        - If 'u', change to 'a'
+      - If not, do nothing
+  - Change all consonants to next consonant in alphabet (loop)
+    - Detect if letter is NOT a vowel 
+      - If NOT a vowel, change to next consonant in alphabet
+        - If next is vowel, check next until consonant, then map that
+        - If 'z', change to 'b'
+      - If vowel, do nothing
+  - Print result
+=end
+
+#
+#
+# METHODS
+#
+#
+
+# Get name to be scrambled
+def get_name()
+  puts "What is your real name?"
+  gets.chomp
```

Comment:
don't use `gets.chomp` in methods.  pass a param instead.  `gets.chomp` is for driver code.